### DISCUSSION TITLE
How do you access system app data folder？

### DISCUSSION THREAD

**Discussion by boan-anbo** [tauri-apps/tauri#4708] [Q&A]
My use case: I want to ship two static external files (JSON) with the app. The user could open a dialog and find the two static files already shown in the picker. 

But I dont know which path I should point the picker to. Is there an api to access the default system data folder reserved for the app, or something like that?

Thanks very much!

**Comment by abisar** [comment]
If it's just a json file you can simply put it next to your html, css, js and read the json file through js [fetch api](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).  [more help](https://stackoverflow.com/questions/49481934/fetching-local-json)

**Reply by boan-anbo** [reply]
Thanks!

**Comment by nwalkewicz** [comment]
Import `path` from `@tauri-apps/api` and use `await path.appDataDir()` (or `path.appDataDir().then(...)`) like so:
```ts
import { path } from '@tauri-apps/api';

(async () => {
  const appDataPath = await path.appDataDir();
  console.log(appDataPath); // C:\Users\<current_user>\AppData\Roaming\<package_name>
})();
```
